src/include/common/constants.h

```diff
@@ -178,6 +178,7 @@ struct ClientConfigDefault {
     static constexpr uint64_t TIMEOUT_IN_MS = 0;
     static constexpr uint32_t VAR_LENGTH_MAX_DEPTH = 30;
     static constexpr bool ENABLE_SEMI_MASK = true;
+    static constexpr bool PROGRESS_BAR = true;
```

Comment:
rename ENABLE_PROGRESS_BAR.

Also let's move ClientConfigDefault to client_config.h